css/style.css

```diff
@@ -0,0 +1,506 @@
+body {
+  font-family: 'GothamPro',  sans-serif;
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line list (value-list-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/Final-Axion-cite/pullRequest?prid=5985384)